src/main/java/bouncybob/task/Deadlines.java

```diff
@@ -0,0 +1,58 @@
+package bouncybob.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task in the BouncyBob application.
+ */
+public class Deadlines extends Task {
```

Comment:
I'd encourage you to avoid plural class names as part of the Java coding standard for this course! 
```suggestion
public class Deadline extends Task {
```